src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -147,6 +145,15 @@ public String getWikiGlobalURL(){
     }
   }
 
+  public String getFandomUrl() {
+    EnvType env = Configuration.getEnvType(EnvType.PROD.getKey());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "env" which hides the field declared at line 14. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AHiddenFieldCheck)